api/geth_backend.go

```diff
@@ -1030,7 +1030,14 @@ func (b *GethStatusBackend) createTempDBFile(pattern string) (tmpDbPath string,
 		err = errors.New("root datadir wasn't provided")
 		return
 	}
-	file, err := os.CreateTemp(filepath.Dir(b.rootDataDir), "*-"+pattern)
+	rootDataDir := b.rootDataDir
+	if !strings.HasSuffix(rootDataDir, "/") {
```

Comment:
Could you please leave a comment explaining the issue?

Reply:
sure